### ISSUE TITLE
Remove formatting modifying rewriting content

### ISSUE THREAD

**Issue by adab4d** [penge/my-notes#362]
I stumbled on the formatting removal feature (accessed either via the downward button or keyboard shortcut) to alter the text I am clearing.

I see text being modified in two way:
1. blank lines are being spaces, only some occurencies and not all
2. lines get duplicated and inserted at a seemingly random place

In order to reproduce:

 create a new note, then open the file with another editor (I use either a local editor with the file synced with Insync or connect the Google Workspace app texteditor.co, both way will keep the file creator allowing My Notes to read the changes) and paste

```html
<div>* P<br />C<br />I</div>
<div>** R<br />- B<br />- C<br />- V</div>
```

This is what I get My Notes to save when I type the content, plain text.

Now select all text and remove formatting, the resulting HTML content in my case has the newlines and a duplicate lines

```html
<div>* P<br><div>C<br>I
<div>** R<br>- B<br>- C<br>- V</div>- B<br>- C<br>- V</div></div><div>** R<br></div>
```

I expect a format removal to only remove formatting, not elements, but I figure dealing with HTML there's a bit of mixup.

The duplication is an error though.

**Comment by penge** [comment]
Now when the RAW mode coming (I plan to release it very soon), I was able to test this quite easily and there was something unexpected going on as you pointed out. In my case, it didn't duplicate things, but reordered them – which isn't ideal too.

Current implementation of **"Remove Formatting"** button was also trying to format the block using `<div>` as a way to revert `<pre>` produced by **"Code"** button. This indeed caused some unexpected results.

I have updated what **"Remove Formatting"** should do, giving it less responsibilities by removing the format only, and the problem was resolved.

I plan to release this fix as part of version 3.22.
